index.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   getSequence
 }
 
- // console.log("Fib Seq: " + getList(process.argv[2]));
+ console.log("Fib Seq: " + getList(process.argv[2]));
```

Comment:
Let's be a little more verbose... 
```suggestion
console.log("Fibonacci Sequence: " + getList(process.argv[2]));